Makefile

```diff
@@ -24,6 +24,7 @@ clean-build:
 	rm -fr *.egg-info
 	rm -fr .eggs
 	rm -fr .cache
+	rm -r kolibri/dist/exercise_perseus_renderer || true
```

Comment:
This workaround gets the job done for now, but we'll need a long-term solution that doesn't reference external plugin names. (For example, the upcoming vodafone plugin.)

Maybe add a comment explaining what's happening?